### ISSUE TITLE
Mutation numbering and B outgroup

### ISSUE THREAD

**Issue by rneher** [blab/nextflu#69]
As of now:
- we clip the signal peptide of H3N2 (ignoring the first 48 nucleotides)
- Our H1N1 starts counting at the methionine (beginning of outgroup), while NIMR pdf start counting at the DTLCIG.... (ignoring the first 51 nucs)
- our B outgroup starts with CRSRAFSNIHKMKAIIVLLMVVTSNA_DRICT, the underscore indicates where NIMR starts counting. that is 26 aa == 78nuc into our outgroup. 

clearly, we want to harmonize this. requires defining the start of the cds and adjusting of clade defining amino acid numbers.

**Comment by rneher** [comment]
this explains it:

http://www.ncbi.nlm.nih.gov/pmc/articles/PMC326080/?page=4

**Comment by rneher** [comment]
done, I guess

**Comment by trvrb** [comment]
I think this is good. I believe it should automatically close once everything is merged into `master`.
